src/SonarScanner.MSBuild.PreProcessor/AnalysisConfigGenerator.cs

```diff
@@ -42,7 +42,8 @@ public static class AnalysisConfigGenerator
         Dictionary<string, string> additionalSettings,
         IDictionary<string, string> serverProperties,
         List<AnalyzerSettings> analyzersSettings,
-        string sonarQubeVersion)
+        string sonarQubeVersion,
+        string resolvedJavaExePath)
```

Comment:
All the parameters are documented but this one. Could you please add  a doc line for it?